department-of-veterans-affairs/va.gov-cms
# VACMS-4970: Create CAP Provide opening hours field.

## Description

See #4970 . 

## Testing done

Manual, behat

## Screenshots

![Create_Vet_Center_Community_Access_Point___VA_gov_CMS](https://user-images.githubusercontent.com/643678/114128662-de942a80-98ca-11eb-8a76-021b287cf4a6.png)


## QA steps

1. As sue.candelier@va.gov 
2. Go to node/add/vet_center_cap
3. See new boolean in screenshot above


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
